pkg/targets/adapter/confluenttarget/adapter.go

```diff
@@ -160,7 +206,7 @@ func (a *confluentAdapter) dispatch(event cloudevents.Event) cloudevents.Result
 		return cloudevents.ResultNACK
 	}
 
-	a.logger.Debugf("Delivered message to topic %s [%d] at offset %v",
+	a.logger.Infof("Delivered message to topic %s [%d] at offset %v",
```

Comment:
yes, I meant to do this. Please do not ask me to put it back to debug because I hate when we log to Debug.

Reply:
Could you elaborate on why do you "hate when we log to Debug"? Logging discussion was open for weeks and as a team we agreed that printing individual events on the default level is not only pointless from the user perspective, but also can be financially dangerous for the production deployments.